std/ascii.d

```diff
@@ -187,7 +187,7 @@ else
   +/
 bool isAlphaNum(dchar c) @safe pure nothrow @nogc
 {
-    return c <= 'z' && c >= '0' && (c <= '9' || c >= 'a' || (c >= 'A' && c <= 'Z'));
+    return isAlpha(c) || isDigit(c);
```

Comment:
E.g., this does look like an improvement in readability.

Reply:
Sure, I can do a PR changing only those things then. I also tried to force inlining for those functions, but DMD is acting weirdly on coverage reports.

Reply:
Forcing inlining is a fairly pointless microptimization unless there is a bug in the optimizer. 

These types of functions are literally like shooting fish in a barrel with a machine gun for a compiler.

Reply:
[Without `pragma(inline, true)`, you don't get cross-module inlining with LDC. So if you don't use LTO (incl. linking the Phobos LTO lib) and don't want to experiment with `-enable-cross-module-inlining`, selective usage of `pragma(inline, true)` for hot little functions can pay off.]

Reply:
https://d.godbolt.org/z/9Pbof7 

I wasn't aware of this limitation, so in that case I am in favour - the only thing worth considering is potentially profiling the optimized dmd build since I have seen some comments suggesting performance issues with inlining (I personally don't really care about the performance of dmd release builds but I guess someone might)